### PR TITLE
Improvements to Navigation Filters and Profile Cards

### PR DESCRIPTION
**Pull request by santisiri**
- Gains space by reducing the height of the search bar and dropdown filter
- Uses a color code to signal the status of an application
- Signals if the profiles comes from Mainnet or other chains (preparing territory for PoH v2)
- Links to [proofofhumanity.eth.limo](https://proofofhumanity.eth.limo)

### CODE REVIEW

File: public/locales/pt/translation.json [hunk 1]
```diff
@@ -247,5 +247,12 @@
   "web3_modal_metamask": "Conecte sua carteira Metamask",
 
   "404_title": "Página não encontrada",
-  "404_descrição": "A página solicitada não existe. Por favor, tente novamente"
+  "404_descrição": "A página solicitada não existe. Por favor, tente novamente",
```
Comment (mmiguerodriguez) [inline public/locales/pt/translation.json]: Shouldn't it be `404_description`?

**Review by mmiguerodriguez — APPROVED** [review]
LGTM. Please fix the minor change in translations before merging

File: public/locales/pt/translation.json (hunk 1 above)
Comment (santisiri) [reply public/locales/pt/translation.json]: Good catch but that certainly wasn't me. Will fix on a subsequent pull (already fixed in the branch i'm currently working on: https://github.com/bilinkis/proof-of-humanity-web/commit/51b9991ea1a6db7307432dc5d211e54982ec16a3).

**Comment by santisiri** [comment]
OK, I've fixed the portuguese transaltion json file and polished the filter dropdown menu look & feel.

**Pull request merged**